bay/containers/container.py

```diff
@@ -138,7 +138,7 @@ def load(self):
                 git_match = self.git_volume_pattern.match(source)
                 if git_match:
                     source = "../{}/{}".format(git_match.group(1), git_match.group(2).lstrip("/"))
-                self._devmodes[name][mount_point] = source
+                self._devmodes[name][mount_point] = os.path.abspath(os.path.join(self.graph.path, source))
```

Comment:
Huh my bad, looks like the builds are red because of https://github.com/eventbrite/bay/blob/master/bay/containers/profile.py#L106.